understanding/22/focus-not-obscured-minimum.html

```diff
@@ -48,14 +53,95 @@ <h2>Intent of Focus Not Obscured (Minimum)</h2>
+
+            <figure id="">
+              <img src="" alt="" />
+              <figcaption>Placeholder for future illustration pair showing an abstract representation of a sidebar reflowing a thinner main content region</figcaption>
+          </figure>
+         </li>
+          <li><strong>When the additional content is disclosed, it takes focus and the tab ring is constrained to the new content until it is dismissed.</strong> This modality is somewhat like a dialog, in that a user cannot navigate beyond the disclosed content by keyboard without dismissing it first (typically by pressing Esc). However, unlike in a modal dialog, in some implementations a pointer user may be able to interact with content outside the disclosed section without dismissing it. Since this pattern potentially creates an inequitable experience between keyboard and pointer users, it should be used cautiously. That said, it does prevent the disclosed content from obscuring the keyboard focus in the main content, and thus should pass this SC. This is described and demonstrated in a short video in the Knowbility article in the reference section, under the section heading <a href="https://knowbility.org/blog/2020/accessible-slide-menus#keep-keyboard-focus-in-the-slide-out-navigation-until-its-closed">Keep keyboard focus in the slide-out navigation until it’s closed</a>.</ br>
+
+            <figure id="">
+              <img src="" alt="" />
+              <figcaption>Placeholder for future illustrations showing an abstract representation of a side panel where the tab ring is restricted to the panel</figcaption>
+          </figure>
+         </li>
+          <li><strong>The disclosure expands into an area of the page containing no other content.</strong> Many pages are designed with wide margins, providing significant white space into which new content can be disclosed. Many chatbots and toasters are designed to 'slide up' into the right unpopulated side of a page. Where authors are careful to ensure content is not obscured at each breakpoint in a responsive design, no obscuring of other operable content need occur.</ br>
+
+            <figure id="">
+              <img src="" alt="" />
+              <figcaption>Placeholder for future illustrations showing an abstract representation of a chat window expanding into empty space on the right side of the screen.</figcaption>
+          </figure>
+         </li>
+          <li><strong>When focus leaves the additional content, the additional content is hidden or collapsed.</strong> This is very similar to patterns discussed previously under Non-persistent disclosed information. A distinguishing factor can be that the user's last point of interaction in the disclosure is preserved (it persists) even though it may be hidden until a user returns. Some trees and left navigation pattterns behave this way. </ br>
```

Comment:
```suggestion
          <li><strong>When focus leaves the additional content, the additional content is hidden or collapsed.</strong> This is very similar to patterns discussed previously under Non-persistent disclosed information. A distinguishing factor can be that the user's last point of interaction in the disclosure is preserved (it persists) even though it may be hidden until a user returns. Some trees and left navigation patterns behave this way. <br />
```